src/FacetedTokenInput.js

```diff
@@ -149,10 +197,105 @@ export default class FacetedTokenInput extends Component {
         { this.props.children }
 
         { showDropDown && this.renderDropdown() }
+
+
+
+        { /* start test code */ }
+
+        { /*
+          the following div needs to be hidden if this code is going into the
+          master branch (see A11Y_HELPER_STYLE)
+          style={ A11Y_HELPER_STYLE }
+          */ }
+        <div
+          id="a11ysupport"
+          role="log"
+          aria-live="polite"
+          aria-relevant="all"
+          aria-atomic={ true }
+          style={ A11Y_HELPER_STYLE }
+        >
+
+        { /*
+          <span id="tagsupport">
+              simply add all the tokens that exist in the input
+
+              <p></p>
+
+              this adds only the last token, it doesn't solve the current
+              problem since the additions count from the second one onwards
+
+              (tokens.length) ? this.a11ySupport(tokens[tokens.length - 1], tokens.length - 1) : ''
+
+              GOOGLE
+              <div aria-live="polite" aria-atomic="true">
+                pierpaolo.ramon aggiunto. Premi backspace per rimuovere il destinatario.
+              </div>
+
+
+              { tokens.map(this.a11ySupport, this) }
+          </span>
+        */ }
+
+          { this.a11ySupport() }
+        </div>
+
+        { /*
+          this adds a string that thanks to aria-describedby present in the
+          input reads out the number of tokens present in the input in addition
+          to the specifics of the input
+          */ }
+        <div id="number_tokens" style={ A11Y_HELPER_STYLE }>
+          { tokens.length }
+          { '  tokens in input' }
+        </div>
+
+        { /* end test code */ }
+
+
+
       </div>
     );
   }
 
+
+  // start experimental method
+
+  // this should add a node that should be detected by aria-live="true"
+  a11ySupport() {
+    const { tokens } = this.state;
+    const { facet, description } = this.props.renderToken(tokens[tokens.length - 1] || []);
+
+    if (tokens.length === tokensLength + 1) {
+      supportText = 'added';
+      tempFacet = facet;
+      tempDescription = description;
+      tempToken[tokens.length - 1] = { facet, description };
+    }
+    else if (tokens.length === tokensLength - 1) {
+      supportText = 'deleted';
+      tempFacet = tempToken[tokens.length].facet;
+      tempDescription = tempToken[tokens.length].description;
+    }
+
```

Comment:
if a token is added I need the support text to be 'added' and I need to store the token in a temporary array.
If a token id removed I need to change the text to 'deleted' and I need to get the facet and description of the deleted token